tensorflow/compiler/xla/literal.cc

```diff
@@ -1891,13 +1894,13 @@ bool LiteralBase::IsR1Iota() const {
   auto is_iota_at_idx = [&](const int64 idx) {
     switch (shape().element_type()) {
       case U8:
-        return Get<uint8>({idx}) == idx;
+        return static_cast<uint8>(Get<uint8>({idx})) == idx;
```

Comment:
Should be cast to `int64`, as that is the type of `idx`

Reply:
updated.